tests/test_orthogonalizers.py

```diff
@@ -132,7 +132,7 @@ def test_multicolumn(self):
                 ),
             )
             self.assertEqual(
-                str(cm.message),
+                str(cm.exception),
```

Comment:
I think here you are missing an indentation

Reply:
Thanks, fixed now.